tests/Unit/IO/Importers/CMakeLists.txt

```diff
@@ -33,20 +33,11 @@ function(add_algorithm_test TEST_NAME DIM)
     "DataStructures;Domain;ErrorHandling;Informer;IO;Options;Parallel;SystemUtilities"
     )
 
-  add_dependencies(test-executables ${EXECUTABLE_NAME})
-
-  add_test(
-    NAME ${TEST_IDENTIFIER}
-    COMMAND ${CMAKE_BINARY_DIR}/bin/${EXECUTABLE_NAME} --input-file
-    ${CMAKE_CURRENT_SOURCE_DIR}/${EXECUTABLE_NAME}.yaml
-    )
-
-  set_tests_properties(
-    ${TEST_IDENTIFIER}
-    PROPERTIES
-    TIMEOUT 5
-    LABELS "integration"
-    ENVIRONMENT "ASAN_OPTIONS=detect_leaks=0")
+add_standalone_test_with_input_file(
+  "${TEST_IDENTIFIER}"
+  UNUSED
+  "${TEST_NAME}${DIM}D"
```

Comment:
My suggestions to the `add_standalone_test` function above would lead to this code here:

```cmake
add_standalone_test(
  "${TEST_IDENTIFIER}"
  EXECUTABLE_NAME ${EXECUTABLE_NAME} # Already exists, so won't be created again
  INPUT_FILE_NAME "${EXECUTABLE_NAME}.yaml"
  )
```